ui/dashboard/src/components/dashboards/charts/Chart/index.tsx

```diff
@@ -432,6 +432,26 @@ const getOptionOverridesForChartType = (
           overrides = set(overrides, "xAxis.name", xAxisTitleValue);
         }
       }
+
+      // X Axis type setting (for timeseries plots)
+      // Valid chart types: column, area, line (bar, donut and pie make no sense)
+      if (["column", "area", "line"].includes(type) && has(properties, "axes.x.type") && properties.axes.x.type === "time") {
```

Comment:
@jreyesr I believe if we're down to this line of code, then `properties.axes.x` definitely exists, so can probably simplify as:

```suggestion
      if (["column", "area", "line"].includes(type) && properties.axes.x.type === "time") {
```

Reply:
Done. I forgot that in JS you can access a nonexistent property, it returns `undefined` :)